packages/renderer/src/lib/ui/KubernetesCheckConnection.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+import { faRefresh } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@podman-desktop/ui-svelte';
+
+import {
+  kubernetesContextsCheckingStateDelayed,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+
+let currentContextName: string | undefined;
+$: currentContextName = $kubernetesContexts.find(c => c.currentContext)?.name;
+
+function refresh(): void {
```

Comment:
issue: AFAIK kubernetesRefreshContextState is an async method so there is a missing await or a missing catch